Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Copy content
+    In order to copy objects
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        And I am viewing the "News Flash" article
+        When I click on "Copy" link on the "Action Bar"
+        And I select the "Destiny" folder in Universal Discovery Widget
+        And I click the "Confirm selection" button
+        Then I see the message "'News Flash' has been successfully copied under 'Destiny'"
```

Comment:
There is one major blocker: those will be translated. 

Yes, it means more implementation, but not that much, in my opinion:

`I am notified that "News Flash" has been copied under "Destination"`

the implementation can then know about the exact sentence pattern it expects to find, and can use a generic method from the Notification context: 

```php
/**
 * @then /^I am notified that :contentName was copied to :destinationName$/
 */
function iAmNotifiedThatContentWasCopiedTo( $contentName, $destinationName )
{
  $notificationMessage = sprintf(
    "'%s' has been successfully copied under '%s'",
    $contentName, $destinationName
  );
  $this->iAmNotifiedThat( $notificationMessage );
}
```

The implementation details (the exact text) is hidden, and the user/scenarist only cares about the fact that he does get notified about the copy of the content under its destination.

I know it adds a bit of code, but frankly not that much, and more importantly: it is *very* simple code. Just more methods, but with almost no complexity.